CMakeLists.txt

```diff
@@ -306,6 +306,11 @@ if(UNIX)
     add_compile_options(-fstack-protector --param ssp-buffer-size=4)
     link_libraries(-fstack-protector --param ssp-buffer-size=4)
   endif()
+
+  check_c_compiler_flag(-fno-common HAVE_FNO_COMMON)
+  if (HAVE_FNO_COMMON)
+    add_compile_options(-fno-common)
+  endif()
```

Comment:
why is this in the `if(UNIX)` block? Is there a reason to avoid it on non-unix, if the flag is supported there?

Reply:
I've moved it out of the UNIX if-clause.